src/medCore/gui/toolboxes/medAbstractToolBox.h

```diff
@@ -17,27 +17,35 @@
 #include <medCoreExport.h>
 
 class medAbstractData;
-class medSegmentationSelectorToolBox;
-class medSegmentationAbstractToolBoxPrivate;
+class medSelectorToolBox;
+class medAbstractToolBoxPrivate;
 
-//! Base class for custom segmentation algoithms
-class MEDCORE_EXPORT medSegmentationAbstractToolBox : public medToolBox
+//! Base class for custom algorithms
+class MEDCORE_EXPORT medAbstractToolBox : public medToolBox
```

Comment:
```medAbstractToolBox``` is too generic. This is a specialisation of ```medToolBox``` for toolboxes that can be used in a selector interface, so it should be called something like ```medAbstractSelectableToolBox```.